<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PromotionWidget</class>
 <widget class="QWidget" name="PromotionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PromotionWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="TitleBarWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>330</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_2 {
	border-image: url(:/shareImages/dlg_background.png);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>22</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>26</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>330</width>
          <height>282</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>22</y>
           <width>13</width>
           <height>14</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>13</width>
           <height>14</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/saleImage/cu.png);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>20</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:16px;
font-family:&quot;微软雅黑&quot;;
color: rgb(255, 121, 121);
font:bold;</string>
         </property>
         <property name="text">
          <string>满100减10</string>
         </property>
        </widget>
        <widget class="QLabel" name="shoppingCartLabel_01">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>46</y>
           <width>80</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(153, 153, 153);
</string>
         </property>
         <property name="text">
          <string>序号：</string>
         </property>
        </widget>
        <widget class="QLabel" name="goodsNameLabel_01">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>46</y>
           <width>100</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);
</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="shoppingCartLabel_02">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>72</y>
           <width>80</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(153, 153, 153);
</string>
         </property>
         <property name="text">
          <string>序号：</string>
         </property>
        </widget>
        <widget class="QLabel" name="goodsNameLabel_02">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>72</y>
           <width>100</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="shoppingCartLabel_03">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>98</y>
           <width>80</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(153, 153, 153);
</string>
         </property>
         <property name="text">
          <string>序号：</string>
         </property>
        </widget>
        <widget class="QLabel" name="goodsNameLabel_03">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>98</y>
           <width>100</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="shoppingCartLabel_04">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>124</y>
           <width>80</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(153, 153, 153);
</string>
         </property>
         <property name="text">
          <string>序号：</string>
         </property>
        </widget>
        <widget class="QLabel" name="goodsNameLabel_04">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>124</y>
           <width>100</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="shoppingCartLabel_05">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>150</y>
           <width>80</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(153, 153, 153);
</string>
         </property>
         <property name="text">
          <string>序号：</string>
         </property>
        </widget>
        <widget class="QLabel" name="goodsNameLabel_05">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>150</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="shoppingCartLabel_06">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>170</y>
           <width>80</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(153, 153, 153);
</string>
         </property>
         <property name="text">
          <string>序号：</string>
         </property>
        </widget>
        <widget class="QLabel" name="goodsNameLabel_06">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>170</y>
           <width>100</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(51, 51, 51);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="promotionStatusLabel_01">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>46</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(117, 170, 86);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="promotionStatusLabel_02">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>72</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(117, 170, 86);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="promotionStatusLabel_03">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>98</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(117, 170, 86);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="promotionStatusLabel_04">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>124</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(117, 170, 86);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="promotionStatusLabel_05">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>150</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(117, 170, 86);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="promotionStatusLabel_06">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>170</y>
           <width>60</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:12px;
font-family:&quot;微软雅黑&quot;;
color: rgb(117, 170, 86);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
